src/Exception/ConfigurationException.php

```diff
@@ -48,6 +49,13 @@ public static function requiresStrategy(
         return new self(self::MSG_STRATEGY_REQUIRED, 0, $previous);
     }
 
+    public static function requiresStatefulness(
```

Comment:
A new message variant for the ConfigurationException type, when a stateful method is incorrectly invoked in a stateless application type.